src/parse.c

```diff
@@ -2103,9 +2103,10 @@ handle_bridge_path_cost(NetplanParser* npp, yaml_node_t* node, const char* key_p
             if (*endptr != '\0')
                 return yaml_error(npp, node, error, "invalid unsigned int value '%s'", scalar(value));
 
-            g_debug("%s: adding path '%s' of cost: %d", npp->current.netdef->id, scalar(key), v);
+            g_debug("%s: adding path '%s' of cost: %ld", npp->current.netdef->id, scalar(key), v);
```

Comment:
question: I wonder if we should be using some marcos from `<inttypes.h>` here instead (e.g. `PRIu64`), as `%ld` is not always guaranteed to match uint64.

I guess we should also try (cross-)compiling this branch for armhf and s390x to see if there are any side-effects.

Reply:
good point, I'll create a package from this branch and test it against all the archs supported by Ubuntu